src/entities/permit2Helper/index.ts

```diff
@@ -358,7 +358,20 @@ const getNonce = async (
         functionName: 'allowance',
         args: [owner, token, spender],
     });
-    const nonce = result[1];
+    const nonce = result[2];
```

Comment:
I think nonce is the third value returned by `allowance`
https://github.com/Uniswap/permit2/blob/cc56ad0f3439c502c246fc5cfcc3db92bb8b7219/src/interfaces/IAllowanceTransfer.sol#L108-L114

Reply:
Looks like you're right! Not sure why this hasn't been an issue until now.

Reply:
I think the v3 FE is fetching the nonce manually instead of letting the SDK grab it